pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -534,7 +547,8 @@ func (r *hibernationReconciler) nodesReady(cd *hivev1.ClusterDeployment, remoteC
 	if hibernatingCondition == nil {
 		return false, errors.New("cannot find hibernating condition")
 	}
-	if time.Since(hibernatingCondition.LastProbeTime.Time) < nodeCheckWaitTime {
+	// Don't delay nodeCheckWaitTime the first time through.
+	if hibernatingCondition.Reason != hivev1.InitializedConditionReason && time.Since(hibernatingCondition.LastProbeTime.Time) < nodeCheckWaitTime {
```

Comment:
check for status unknown instead?

Reply:
Done.

Reply:
This is now checking for the new SyncSetsApplied reason instead.